harness/determined/cli/tensorboard.py

```diff
@@ -29,12 +29,19 @@ def start_tensorboard(args: Namespace) -> None:
 
     req_body["files"] = context.read_legacy_context(args.context, args.include)
 
-    resp = api.post(args.master, "api/v1/tensorboards", json=req_body).json()["tensorboard"]
+    api_resp = api.post(args.master, "api/v1/tensorboards", json=req_body).json()
```

Comment:
if you are going to touch an `api.post()`, please update it to use the generated bindings.

Reply:
updated